Backend/router/v1/report.ctrl.js

```diff
@@ -10,32 +10,42 @@ const createReport = (req, res) => {
   }).then((result) => {
     res.status(201);
     res.send("good");
+  }).catch((err) => {
+    res.json({msg: "error"});
   });
 }
 
 const showReportList = (req, res) => {
   models.report.findAll().then((result) => {
     res.json(result);
-  }).catch((err) => {});
+  }).catch((err) => {
+    res.json({msg: "error"});
+  });
 }
 
 const findReport = (req, res) => {
-  const reportId = parseInt(req.params.reportId)
+  const reportId = parseInt(req.query.reportId);
   //When ID is not a number
```

Comment:
Not necessary, code is pretty much self-explanatory

Reply:
Do you mean comment or using query? @shberry